.travis.yml

```diff
@@ -9,3 +9,4 @@ env:
     - TWILIO_ACCOUNT_SID=ACYour-Account-SID
     - TWILIO_AUTH_TOKEN=Your-Twilio-Authy-Token
     - TWILIO_NUMBER=Your-Twilio-Phone-Number
+    - TWIML_APPLICATION_SID=Your-Twiml-Application_SID
```

Comment:
I think this needs to add: 
```yml
node_js:
  - 'stable'
  - '4.2'
```
And if you change that you can also drop:
```yml
install:
- npm install
```
since its not needed.